### ISSUE TITLE
Wish list for 2024 dev

### ISSUE THREAD

**Issue by jpolton** [NOC-MSM/pyBDY#172]

**Comment by katavouta** [comment]
Yes an arbitrary named tracer at grid T that we can control from the namelist (since a scalar that will follow exactly what is done for T and S so it should be easy) and is written to each own bdy file. This can be used to generate OBC for the biogeochemistry etc.
